infra/build/functions/main.py

```diff
@@ -39,3 +41,10 @@ def build_base_images(event, context):
 def coverage_build(event, context):
   """Entry point for cloud function to build coverage reports."""
   request_coverage_build.request_coverage_build(event, context)
+
+
+def introspector_build(event, context):
+  """Entry point for cloud function to build introspector reports."""
+  request_introspector_build.request_introspector_build(event, context)
```

Comment:
You need to add this file too (request_introspector_build.py).

Reply:
Ah! done...